lua/rest-nvim/config/init.lua

```diff
@@ -126,6 +126,11 @@ local default_config = {
         end,
       },
     },
+    keybinds = {
+      buffer_local = false,
+      prev = "H",
+      next = "L",
+    },
```

Comment:
Could you please annotate this configuration options table? The rest of the annotations are at the top of the file, in case you need some examples and where to place them

Reply:
[Done](https://github.com/rest-nvim/rest.nvim/pull/351/commits/e2e77cd50fcb0946126b9c35b9434d937ac2706a)